fe/fe-core/src/test/java/com/starrocks/sql/parser/ParserTest.java

```diff
@@ -244,6 +244,22 @@ void testParseLargeDecimal() {
         }
     }
 
+    @Test
+    void testDecimalTypeDeclarationMysqlCompatibility() {
+        String sql = "select cast(1 as decimal(65)),cast(1 as decimal)";
```

Comment:
is 65 valid? Is the maximum precision 39?

Reply:
precision > 38 is supported now, use double/decimal as underlying type.